scripts/quests/otherAreas/Waking_The_Beast.lua

```diff
@@ -64,12 +64,12 @@ quest.sections =
             onEventFinish =
             {
                 [208] = function(player, csid, option, npc)
-                    player:setCharVar("WTB_CONQUEST", getConquestTally())
+                    quest:setVar(player, 'conquest', getConquestTally())
```

Comment:
these vars should follow the rules of use for Prog, Stage, Option - i would use option
quest:setvar(player, 'Option', getConquest Tally())